config/deploy.rb

```diff
@@ -44,11 +44,12 @@
       end
     end
   end
-  
+
   desc "Restart zartan-related monitored processes"
   after :seed_config, :restart do
     on roles(:web) do
-      if test("ps cax | grep monit")
+      enabled_file = File.join(fetch(:deploy_to), 'shared', 'enabled')
```

Comment:
No automated process creates this file. It serves to prevent restarting monit in the case where we need to preform an initial deploy to a machine where monit is installed and running, but not configured to run Zartan services.